reana-ui/src/components/TopHeader.js

```diff
@@ -12,31 +12,25 @@ import { Image, Icon, Popup, List } from "semantic-ui-react";
 import { useDispatch, useSelector } from "react-redux";
 import { Link } from "react-router-dom";
 
-import { getConfig, getUserEmail } from "~/selectors";
+import { getUserEmail } from "~/selectors";
 import LogoImg from "~/images/logo-reana.svg";
 import { userSignout } from "~/actions";
+import NotificationIcon from "./NotificationIcon";
 
 import styles from "./TopHeader.module.scss";
 
 export default function TopHeader() {
   const dispatch = useDispatch();
   const email = useSelector(getUserEmail);
-  const config = useSelector(getConfig);
-  const signOut = () => {
-    dispatch(userSignout());
-  };
+  const signOut = () => dispatch(userSignout());
 
   return (
     <header className={styles["top-header"]}>
       <Link to="/">
         <Image src={LogoImg} size="small" />
       </Link>
       <section>
-        {config.docsURL && (
-          <a href={config.docsURL} target="_blank" rel="noopener noreferrer">
-            <Icon name="question circle outline"></Icon> Help
-          </a>
-        )}
+        <NotificationIcon />
```

Comment:
Yeah, it's very hard to come up with a name for this one. I think adding "old" or "new" might also be confusing, since it will not replace the `Notification` component. Also adding "quota" is not correct, since later it will be used for interactive sessions, etc. I've changed it to `NotificationBell` for now since I think this is a standard way to name this thing.. Let me know if you can come up with something more clear